src/script/page/MainContent/panels/preferences/OptionPreferences.tsx

```diff
@@ -103,7 +103,7 @@ const OptionPreferences: React.FC<OptionPreferencesProps> = ({
   };
 
   const saveOptionNewTheme = (useDarkMode: boolean) => {
-    const newTheme = useDarkMode ? ThemeViewModelThemes.DARK : ThemeViewModelThemes.DEFAULT;
+    const newTheme: Theme = useDarkMode ? 'dark' : 'default';
```

Comment:
This looks like a random string, but it's actually autocompleted and typechecked by typescript. No need for `const` for that anymore